pyseir/cli.py

```diff
@@ -376,6 +393,7 @@ def map_outputs(state, output_interval_days, run_mode, states_only):
 @click.option(
     "--skip-whitelist", default=False, is_flag=True, type=bool, help="Skip the whitelist phase.",
 )
+@click.option("--fips", help="County level fips code to restrict runs to.")
```

Comment:
Maybe worth noting (1) that those fips can be across any state and (2) --states-only flag will override this and (3) I think you still have to build all for states with the current flow. So passing a single fips will still run for all states (and then just one county). Passing the -s flag will limit to one state and one county (not enforced to be the same) but if you had that in the doc that seems reasonable.

Reply:
Yeah that's a good point.  There's not a ton of guidance around how to actually use it.  I'll add that to the help string, thanks!

Reply:
I added to this comment, let me know if that's more clear or there's a better way to phrase it